composer.json

```diff
@@ -73,11 +73,9 @@
     "scripts": {
         "symfony-scripts": [
             "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
```

Comment:
https://github.com/symfony/symfony-standard/blob/master/web/app.php#L8, opcache does the job, it adds overhead for no gain.